perfmetrics/scripts/testing_on_gke/examples/utils/utils.py

```diff
@@ -136,5 +143,190 @@ def unix_to_timestamp(unix_timestamp: int) -> str:
   return utc_timestamp_string
 
 
-def standard_timestamp(timestamp: int) -> str:
+def standard_timestamp(timestamp: str) -> str:
   return timestamp.split(".")[0].replace("T", " ") + " UTC"
+
+
+def timestamp_to_epoch(timestamp: str) -> int:
+  return int(
+      time.mktime(
+          time.strptime(
+              timestamp.split(".")[0].replace("T", " "), "%Y-%m-%d %H:%M:%S"
+          )
+      )
+  )
```

Comment:
See if this works.
```suggestion
  return int(datetime.strptime('2024-08-21T19:20:25.444', "%Y-%m-%dT%H:%M:%S.%f").timestamp())
```

Reply:
yes it works, with a little modification.